.circleci/config.yml

```diff
@@ -33,6 +38,14 @@ jobs:
       - run:
           name: Smoke test Echo API
           command: |
-            dockerize -wait tcp://0.0.0.0:3000 -timeout 1m
+            dockerize -wait tcp://localhost:16686 -timeout 1m # wait for jaeger
+            dockerize -wait tcp://0.0.0.0:3000 -timeout 1m # wait for echo-api
+
             curl --fail http://0.0.0.0:3000/test
 
+      - run:
+          name: Verify OpenTracing
+          command: |
+            sleep 1 # that is the flush timeout
```

Comment:
A little bit "optimistic" no? 5s to avoid failing tests ?

Reply:
The flush interval is set to 1s: https://github.com/3scale/echo-api/pull/62/files#diff-1b2fd70e37cefee9c4ebf4bc53ca53ecR33
So that should be enough.